storage/storage.cpp

```diff
@@ -768,18 +788,31 @@ void Storage::OnServerListDownloaded(vector<string> const & urls)
   if (m_queue.empty())
     return;
 
-  QueuedCountry const & queuedCountry = m_queue.front();
-  TCountryId const & countryId = queuedCountry.GetCountryId();
-  MapOptions const file = queuedCountry.GetCurrentFile();
+  QueuedCountry & queuedCountry = m_queue.front();
+  if (queuedCountry.GetInitOptions() == MapOptions::Diff)
+  {
+    using diff_scheme::Status;
+    if (m_diffSchemeStatus == Status::Undefined)
+    {
+      m_defferedUpdateTask =
```

Comment:
Why? We just get urls, why do we need to asynchronously call OnServerListDownloaded again?